ewatercycle/models/marrmot.py

```diff
@@ -261,11 +254,8 @@ def parameters(self) -> Iterable[Tuple[str, Any]]:
             ('maximum_soil_moisture_storage', self._parameters[0]),
             ('initial_soil_moisture_storage', self.store_ini[0]),
             ('solver', self.solver),
-            ('start time', self.forcing_start_time.isoformat()),
-            ('end time', self.forcing_end_time.isoformat()),
+            ('start time', self.forcing_start_time.strftime("%Y-%m-%dT%H:%M:%SZ")),
```

Comment:
Looks a bit weird to have a format string, however to get UTC timezone this is needed. Also `self.forcing_*_time` gets sets using get_time which forces UTC so OK to hard-code timezone here.